containers/compose/settings.py

```diff
@@ -0,0 +1,12 @@
+SECRET_KEY = "aabbcc"
+CONTENT_ORIGIN = "http://localhost:24816"
```

Comment:
The user always needs to be able to specify the hostname.

Also, the port would be handled by the webserver.

Reply:
this is the user's file, everything under compose dir is a volume that the user may change or not